features/step_definitions/project.rb

```diff
@@ -164,7 +164,8 @@
     | project_name  | #{project_name}       |
     | node_selector | #{project_name}=label |
     })
-  step %Q/I select a random node's host/
+  step %Q/I store the ready and schedulable workers in the :nodes clipboard/
+  step %Q/I use the "<%= cb.nodes[0].name %>" node/
```

Comment:
@qinpingli since the original step selects a random node, I think you should do the same with your work around.  Use the `sample` method instead of always selecting the first element

Reply:
@pruan-rht Thank you, good catch! I removed the step `I use the <node.name> node`.